tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/config/EnvironmentConfigSource.java

```diff
@@ -0,0 +1,53 @@
+package com.dotcms.config;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.Set;
+
+/**
+ * This class is used to load the properties from the application.properties file.
+ * It is need it to inject the properties into a static block.
```

Comment:
In this case is needed to load properties into a Static block code, because you can't use the traditional CDI through the ConfigProperty annotation, so you can use the Quarkus Config system to read the configuration values. Quarkus provides support for custom ConfigSources that can be used to read additional configuration values and add them to the Config instance in a defined ordinal.

Reply:
need**ed** . There is a typo